File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/accountintegraldetail.js
                                                                export default function area() {
  return {
    current: 1,
    size: 20,
    isNewRecord: false,
    query: {
    // 自增主键
  accountIntegralDetailId: undefined,
      // 账户号
  accountId: undefined,
      // 客户编号
  custId: undefined,
      // 证件类型
  identyType: undefined,
      // 证件号码
  identyCard: undefined,
      // 交易积分
  changeIntegral: undefined,
      // 交易日期
  changeDate: undefined,
      // 积分有效期余额
  integralValidBlance: undefined,
      // 积分有效期
  integralValidDate: undefined,
      // 账户余额
  accountBlance: undefined,
      // 业务
  business: undefined,
      // 描述
  remark: undefined,
      // 积分类型
  integralType: undefined,
      // 预留字段一
  reserveColumn1: undefined,
      // 预留字段二
  reserveColumn2: undefined,
      // 预留字段三
  reserveColumn3: undefined,
          },
    rules: {
        accountIntegralDetailId: [{
  required: true,
  message: '请输入自增主键',
  trigger: 'blur'
  }],
          accountId: [{
  required: true,
  message: '请输入账户号',
  trigger: 'blur'
  }],
          custId: [{
  required: true,
  message: '请输入客户编号',
  trigger: 'blur'
  }],
          identyType: [{
  required: true,
  message: '请输入证件类型',
  trigger: 'blur'
  }],
          identyCard: [{
  required: true,
  message: '请输入证件号码',
  trigger: 'blur'
  }],
          changeIntegral: [{
  required: true,
  message: '请输入交易积分',
  trigger: 'blur'
  }],
          changeDate: [{
  required: true,
  message: '请输入交易日期',
  trigger: 'blur'
  }],
          integralValidBlance: [{
  required: true,
  message: '请输入积分有效期余额',
  trigger: 'blur'
  }],
          integralValidDate: [{
  required: true,
  message: '请输入积分有效期',
  trigger: 'blur'
  }],
          accountBlance: [{
  required: true,
  message: '请输入账户余额',
  trigger: 'blur'
  }],
          business: [{
  required: true,
  message: '请输入业务',
  trigger: 'blur'
  }],
          remark: [{
  required: true,
  message: '请输入描述',
  trigger: 'blur'
  }],
          integralType: [{
  required: true,
  message: '请输入积分类型',
  trigger: 'blur'
  }],
          reserveColumn1: [{
  required: true,
  message: '请输入预留字段一',
  trigger: 'blur'
  }],
          reserveColumn2: [{
  required: true,
  message: '请输入预留字段二',
  trigger: 'blur'
  }],
          reserveColumn3: [{
  required: true,
  message: '请输入预留字段三',
  trigger: 'blur'
  }],
      }
  }
}

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/AccountIntegralDetail.java
package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import org.springclouddev.core.mp.base.BaseEntity;

/**
 * 账户积分交易明细表实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_account_integral_detail")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "AccountIntegralDetail对象", description = "账户积分交易明细表")
public class AccountIntegralDetail extends BaseEntity {

    private static final long serialVersionUID = 1L;

    /**
     * 自增主键
     */
    @ApiModelProperty(value = "自增主键")
    @TableId(value = "account_integral_detail_id", type = IdType.NONE)
    @JsonSerialize(using = ToStringSerializer.class)
    private Long id;
    /**
     * 账户号
     */
    @ApiModelProperty(value = "账户号")
    @TableField("ACCOUNT_ID")
    private String accountId;
    /**
     * 客户编号
     */
    @ApiModelProperty(value = "客户编号")
    @TableField("CUST_ID")
    private String custId;
    /**
     * 证件类型
     */
    @ApiModelProperty(value = "证件类型")
    @TableField("IDENTY_TYPE")
    private String identyType;
    /**
     * 证件号码
     */
    @ApiModelProperty(value = "证件号码")
    @TableField("IDENTY_CARD")
    private String identyCard;
    /**
     * 交易积分
     */
    @ApiModelProperty(value = "交易积分")
    @TableField("CHANGE_INTEGRAL")
    private String changeIntegral;
    /**
     * 交易日期
     */
    @ApiModelProperty(value = "交易日期")
    @TableField("CHANGE_DATE")
    private String changeDate;
    /**
     * 积分有效期余额
     */
    @ApiModelProperty(value = "积分有效期余额")
    @TableField("INTEGRAL_VALID_BLANCE")
    private String integralValidBlance;
    /**
     * 积分有效期
     */
    @ApiModelProperty(value = "积分有效期")
    @TableField("INTEGRAL_VALID_DATE")
    private String integralValidDate;
    /**
     * 账户余额
     */
    @ApiModelProperty(value = "账户余额")
    @TableField("ACCOUNT_BLANCE")
    private String accountBlance;
    /**
     * 业务
     */
    @ApiModelProperty(value = "业务")
    @TableField("BUSINESS")
    private String business;
    /**
     * 描述
     */
    @ApiModelProperty(value = "描述")
    private String remark;
    /**
     * 积分类型
     */
    @ApiModelProperty(value = "积分类型")
    @TableField("INTEGRAL_TYPE")
    private String integralType;
    /**
     * 预留字段一
     */
    @ApiModelProperty(value = "预留字段一")
    @TableField("RESERVE_COLUMN1")
    private String reserveColumn1;
    /**
     * 预留字段二
     */
    @ApiModelProperty(value = "预留字段二")
    @TableField("RESERVE_COLUMN2")
    private String reserveColumn2;
    /**
     * 预留字段三
     */
    @ApiModelProperty(value = "预留字段三")
    @TableField("RESERVE_COLUMN3")
    private String reserveColumn3;


}
